digital-credentials/get.tentative.https.html

```diff
@@ -28,6 +28,23 @@
     ]);
   });
 
+  promise_test(async (t) => {
+    const invalidData = [null, undefined, "", 123, true, false];
+    for (const data of invalidData) {
+      const options = makeGetOptions([
+        {
+          data,
+          protocol: "openid4vp",
+        },
+      ]);
+      await promise_rejects_js(
+        t,
+        TypeError,
+        navigator.credentials.get(options)
+      );
+    }
```

Comment:
Are you sure this actually work?
The documentation says 
```
 @param {string | string[]} [requestsToUse] - Request types ('default', 'openid4vp', or an array). Defaults to ['default'].
```
I don't think it is meant to be used this way?
I looked at the code, and it seems to be legit actually.


```suggestion
    for (const d of invalidData) {
       const options = {
       digital: {
         requests: [
               {
                    data: d,
                   protocol: "openid4vp",
              },
          ]
      }};
      await promise_rejects_js(
        t,
        TypeError,
        navigator.credentials.get(options)
      );
    }
```

Reply:
for ` for (const d of invalidData) {` we can probably just do ` for (const data of invalidData) {`